### PR TITLE
Add a blog

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
   },
   "scripts": {
     "cleanup": "del-cli db.json",
-    "start": "hexo generate --config docs_config.yml --watch & node server.js & wait",
+    "start": "hexo clean && hexo generate --config docs_config.yml,blog_config.yml --watch & node server.js & wait",
```
Comment (phated) [inline package.json:10]: Should `cleanup` be changed to `hexo clean` and then we can add it to `prestart` like `prebuild`?

File: themes/grain/layout/index.ejs [hunk 2]
```diff
@@ -1,59 +1,51 @@
...
+      <% }); %>
+    </div>
+    <div class="blog-sidebar">
+      <h1>The Grain Blog</h1>
+      <p class="blog-subtitle">Your source for everything Grain.</p>
+      <hr>
+      <p class="blog-subtitle">Like what we're doing with Grain? <a href="https://twitter.com/grain_lang">Follow us on Twitter</a> or come <a href="https://discord.gg/7U3newJ">chat with us on Discord</a>.</p>
+      <div class="join-mailing">
+        <h4 class="join-mailing-title">Join our mailing list for periodic farm-to-desktop goodness:</h4>
+        <!-- Begin Mailchimp Signup Form -->
```
Comment (phated) [inline themes/grain/layout/index.ejs:30]: 🔥 🔥 🔥 🔥

File: themes/grain/scripts/authors.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  'Oscar Spencer': {
```
Comment (phated) [inline themes/grain/scripts/authors.js:2]: I like this config! 👍

**Comment by ospencer** [comment]
Closes #83

**Pull request merged**